mgmt/src/processor/tests.rs

```diff
@@ -257,6 +258,7 @@ pub mod test {
     }
 
     #[test]
+    #[traced_test]
```

Comment:
What does this do?

Reply:
This enables tracing on the test. Else, the debug!, warn! etc. aren't shown. This only matters if you run the tests with --nocapture, which I often do to see if the logs we add are meaningful.